js/restaurant.js

```diff
@@ -414,6 +431,22 @@ function fireRule(rule) {
   }
 
   if(win){
+    if (progress.guessHistory[currentLevel]) {
+      if (progress.guessHistory[currentLevel].solutions) {
+        if (!progress.guessHistory[currentLevel].solutions.includes(rule)) {
+          $(".display-help .solutions-title").show();
```

Comment:
Actually this line is redundant since if we have solutions the title should have been shown